install_config/configuring_sdn.adoc

```diff
@@ -289,19 +288,22 @@ service neutron-vpn-agent restart
 service neutron-server  restart
 ----
 
-When creating the OpenStack instances, disable both port-security and security
-groups in the ports where the container network Flannel interface will be:
-
+. When creating the {product-title} instances, disable both port security and security
```

Comment:
Should this be changed? Our product title is not OpenStack?

Reply:
Maybe:
"When creating the {product-title} instances in OpenStack,..."